va/va_cp_vp9.h

```diff
@@ -0,0 +1,121 @@
+ *
+ */
+typedef struct _VACencHeaderParameterBufferVP9 {
+    
+    union {
+        struct {
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t profile                              : 2;            
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t show_existing_frame_flag             : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t show_to_show                         : 3;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t frame_type                           : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t show_frame                           : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t error_resilient_mode                 : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t color_space                          : 2; 
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t subsampling_x                        : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t subsampling_y                        : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t refresh_frame_flags                  : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t frame_parallel_decoding_mode         : 1;
+            /** \brief Same as the VP9 bitstream syntax element. */
+            uint32_t render_and_frame_size_different      : 1;
+            /** \brief Reserved for future use, must be zero */
+            uint32_t reserved                             : 20;
+            } bits;
+        uint32_t value;
+    } header_fields; 
+     /** \brief Same as the VP9 bitstream syntax element. */
+    uint16_t frame_width_minus1;
+     /** \brief Same as the VP9 bitstream syntax element. */
+    uint16_t frame_height_minus1;
+    /** \brief Same as the VP9 bitstream syntax element. */
+    uint16_t render_width_minus1;
+    /** \brief Same as the VP9 bitstream syntax element. */
+    uint16_t render_height_minus1;
+    /** \brief Reserved bytes for future use, must be zero */
+    uint32_t va_reserved[VA_PADDING_MEDIUM];
+} VACencHeaderParameterBufferVP9;
+
+
+/**
+ * \brief cenc status report data for VP9 cenc decode
+ *
+ * This structure holds information for cenc decode status 
+ * of data in VACencStatusBuf
+ *
+ */
+typedef struct _CencStatusReportVp9 {
+    /* \brief encryption status defined by VAEncryptionStatus */
+    uint32_t                       status;
+    /* \brief status report feed back number */
+    uint16_t                       status_report_feedback_number;
+    /** \brief header parameter*/
+    VACencHeaderParameterBufferVP9 header_info;
```

Comment:
Is it defined that encrypted VP9 streams must always have the superframe index in the clear (so the user decomposes the stream into frames and passes them separately), or must not be encapsulated in superframes?

Reply:
When encrypting superframes,  the the superframe header must be clear. Defined in VP Codec ISO Media File Format Binding.